src/app/tab/covid/covid.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="covidContainer" [style.padding]="getWidth(settings.config.covidData.countries[index].padding)">
+  <div class="flag" *ngIf="settings.config.covidData.countries[index].enableFlag"
+  >
+    <img  [ngStyle]="{ 'height' :shared.getImageSize(settings.config.covidData.countries[index].flagSize)}"  [src]="countryData.countryInfo.flag" alt="flag" />
+  </div>
+  <div class="info" [style.padding]="getWidth(settings.config.covidData.countries[index].padding)">
+    <div><strong>Country :</strong> {{ countryData.country }}</div>
```

Comment:
Don't bold any text. The user has the option of using bolded or regular fonts, so this hard coded bold will interfere with that. No need to differentiate the labels.